roles/setup_http_store/tasks/main.yml

```
@@ -6,7 +6,15 @@
         name: podman
         state: present
       become: true
-
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)